### DISCUSSION TITLE
Download marked as harmful by chrome, virustotal declares it as virus.

### DISCUSSION THREAD

**Discussion by HochdruckHummer** [magicbug/Cloudlog#2930] [Aurora]
I tried to download aurora, but it was marked as harmful by Chrome. That made me suspicious. So I uploaded it to Virustotal which also identified it with different malicious backgrounds.

I downloaded it here: https://aurora.cloudlog.org

Is this message valid, false positive or did I use a wrong download page?

**Comment by magicbug** [comment]
Correct page sadly its being classed as bad due to having no code-signed certificate as I need to spend a $200 on one.

**Reply by HochdruckHummer** [reply]
Thanks! Maybe put that to the website as an information?

By the way: Thank you very much for CLoudlog.

Yesterday I tried to install it on a webhoster... No chance, because directory was not drawn correctly I guess from hosting space, as I started the installation already from the external domain. Then tried with Microsoft IIS... Same thing. 

I guess installation routine demands installation on localhost and is not able to set paths accordinbgly when triggered from external domain right at the beginning. Because after installation login wasn't possible due to corrupted uRL-call and images where also not shown. Something must have been wrong with the paths. 
If you ever need one person testing with ext. domain, IIS or hoszting platform, give me a sign.

Then Installed WAMP. Had to mess with Apache for some time to get it working with external domain an referencing A-record... Apache and me will never be friends... Anyways, somehow I managed to get it running. Got my first 18 QSOs (indeed my first ones, not only with CLoudlog) and I love it!
